ets-ecole-de-technologie-superieure.csl

```diff
@@ -122,21 +122,28 @@
     <text variable="issue" prefix=" "/>
   </macro>
   <macro name="access">
-    <group delimiter=". ">
-      <text variable="URL" prefix="&lt;" suffix="&gt;"/>
-      <choose>
-        <if variable="URL" match="any">
-          <group delimiter=" ">
-            <text value="Consulté le"/>
-            <date variable="accessed">
-              <date-part name="day" suffix=" "/>
-              <date-part name="month" suffix=" "/>
-              <date-part name="year"/>
-            </date>
-          </group>
-        </if>
-      </choose>
-    </group>
+		<choose>
```

Comment:
we'd much rather you have the style with two spaces indenting rather than tabs. (All styles on the repo follow this convention). If you're preferred editor doesn't do that for you, we actually have a formatter tool that will do it for you:
http://formatter.citationstyles.org/